packages/block-library/src/query-pagination-numbers/query-pagination-numbers-mid-size-control.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { RangeControl } from '@wordpress/components';
+
+export function QueryPaginationNumbersMidSizeControl( { value, onChange } ) {
+	return (
+		<RangeControl
```

Comment:
Super small nit: We don't have any other controls for this block(and probably don't expect much in the future), so we could easily just add the code in the block's edit function.